From: Daniil Tatianin <99danilt@gmail.com>
Date: Thu, 21 Mar 2024 21:57:12 +0300
Subject: ACPICA: events/evgpeinit: don't forget to increment registered GPE
 count
Patch-mainline: v6.10-rc1
Git-commit: a210accc067ad18d55ea19b6d7a9d9176d809a74
References: jsc#PED-11164

ACPICA commit ba8a36b5c7343cb56af6b331362e97b25e898eb2

This was used to log the number of newly discovered GPEs post table
load in acpi_ev_update_gpes(), but we never incremented the number inside
acpi_ev_match_gpe_method(), so that was never logged.

Link: https://github.com/acpica/acpica/commit/ba8a36b5
Signed-off-by: Daniil Tatianin <99danilt@gmail.com>
Signed-off-by: Rafael J. Wysocki <rafael.j.wysocki@intel.com>
Acked-by: Chun-Yi Lee <jlee@suse.com>
---
 drivers/acpi/acpica/evgpeinit.c |    1 +
 1 file changed, 1 insertion(+)

--- a/drivers/acpi/acpica/evgpeinit.c
+++ b/drivers/acpi/acpica/evgpeinit.c
@@ -413,6 +413,7 @@ acpi_ev_match_gpe_method(acpi_handle obj
 	gpe_event_info->flags &= ~(ACPI_GPE_DISPATCH_MASK);
 	gpe_event_info->flags |= (u8)(type | ACPI_GPE_DISPATCH_METHOD);
 	gpe_event_info->dispatch.method_node = method_node;
+	walk_info->count++;
 
 	ACPI_DEBUG_PRINT((ACPI_DB_LOAD,
 			  "Registered GPE method %s as GPE number 0x%.2X\n",
